docs/controls/risk_assessment/ra-5-screenshots/screenshot

```diff
@@ -0,0 +1 @@
+
```

Comment:
Looks like we can remove this file.

Reply:
file removed

Reply:
Hmm, still seeing it 😕

Reply:
@rnafarrete Can you try to remove the file again?

Reply:
File was deleted.